aten/src/ATen/native/cuda/ScatterGatherKernel.cu

```diff
@@ -2,6 +2,7 @@
 
 #include <ATen/ATen.h>
 #include <ATen/Dispatch.h>
+#include <ATen/MemoryOverlap.h>
```

Comment:
We don't need to add this include for `.set_check_mem_overlap(false)` to work

Reply:
I've moved this into the correct PR.